config/initializers/paperclip.rb

```diff
@@ -8,3 +8,15 @@
   csv: ['application/octet-stream', 'text/plain']
 }
 Paperclip::DataUriAdapter.register
+Paperclip::Attachment.default_options[:storage] = :fog
+Paperclip::Attachment.default_options[:fog_directory] = ENV["GOOGLE_STORAGE_BUCKET_NAME"] || ''
+Paperclip::Attachment.default_options[:path] = ":rails_root/public/system/system/:class/:attachment/:id_partition/:style/:filename"
+Paperclip::Attachment.default_options[:fog_credentials] = {
+    provider: ENV["FOG_PROVIDER"] || "Local",
+    local_root: "#{Rails.root}/public",
+    google_project: 'public-lab' ,
+    google_json_key_location: ENV["GOOGLE_JSON_KEY_FILE"] || '' ,
```

Comment:
Removed the XML keys below this line after input from folks at https://github.com/fog/fog-google/issues/537#issuecomment-877544092

```ruby
    google_storage_access_key_id: ENV["GOOGLE_STORAGE_KEY"] || '' ,
    google_storage_secret_access_key: ENV["GOOGLE_STORAGE_SECRET"] || ''
```